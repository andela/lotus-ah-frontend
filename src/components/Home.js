// react libraries
import React, { Component } from 'react';

// Moduler importations
import Header from './Header';

/**
 * @desc renders home page
 */
class Home extends Component {
  render() {
    return (
      <div>
        <Header />
        <h1>Welcome to Author's Haven, Everyone</h1>
      </div>
    );
  }
}

export default Home;
