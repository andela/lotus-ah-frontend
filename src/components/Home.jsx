// react libraries
import React from 'react';

// components
import Header from './Header';

/**
 * @desc renders home page
 * @returns {object} home
 */
const Home = () => (
      <div>
        <Header />
        <h1>Welcome to Authors Haven, Everyone</h1>
      </div>
);

export default Home;
