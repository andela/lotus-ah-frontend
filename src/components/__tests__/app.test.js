// React Libraries
import React from 'react';

// third-party libraries
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

// modules importation
import Home from '../Home';

configure({ adapter: new Adapter() });

// SnapShot Test
describe('App', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Home debug />);
    expect(component).toMatchSnapshot();
  });
});
