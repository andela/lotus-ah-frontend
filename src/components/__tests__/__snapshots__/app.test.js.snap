// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly in "debug" mode 1`] = `
<div>
  <h1>
    Welcome to Authors Haven, Everyone
  </h1>
</div>
`;
